ingress/controllers/nginx/Dockerfile

```diff
@@ -14,10 +14,12 @@
 
 FROM gcr.io/google_containers/nginx-slim:0.8
 
-RUN apt-get update && apt-get install -y \
+RUN DEBIAN_FRONTEND=noninteractive apt-get update && apt-get install -y \
   diffutils \
+  ssl-cert \
   --no-install-recommends \
-  && rm -rf /var/lib/apt/lists/*
+  && rm -rf /var/lib/apt/lists/* \
+  && make-ssl-cert generate-default-snakeoil --force-overwrite
```

Comment:
@bprashanth I'm sorry, it's valid for 10 years
https://gist.github.com/aledbf/a46eecbf7500ffb8124c30ae2b635cf3#file-gistfile1-txt-L106